### PR TITLE
Feat/controller orders

### PR DESCRIPTION
**Pull request by ElisamaSS**

### CODE REVIEW

File: src/app/controller/OrderController.js [hunk 1]
```diff
@@ -4,101 +4,103 @@ import Product from '../models/Product';
...
         }
-        const orders = ["123", "568"];
+        const orders = [];
 
         async function products(item) {
             if (item > 0) {
-                const colunaTabelaUsario = + item
-                console.log("codigo", colunaTabelaUsario);
-
+                const colunaTabelaUsario = +item;
```
Comment (KarolineLima) [inline src/app/controller/OrderController.js]: remover linha

File: src/app/controller/OrderController.js [hunk 2]
```diff
@@ -4,101 +4,103 @@ import Product from '../models/Product';
...
-
                 const store = await Store.findOne({
-                    where: { id_product: id_product, id_product: true, $and: { id_store: id_store, id_store: true } }
+                    where: {
+                        id: store_id,
+                    },
                 });
 
-                console.log(product, store)
+                console.log(product, store);
```
Comment (KarolineLima) [inline src/app/controller/OrderController.js]: remover linha

File: src/app/controller/OrderController.js [hunk 3]
```diff
@@ -4,101 +4,103 @@ import Product from '../models/Product';
...
+                                optionPickup,
+                                shipping,
+                                orderTotal,
+                                status,
+                                id_store,
+                                id_user,
+                            } = await Order.create(req.body);
+                            Product.update({
+                                quantity: product.quantity - 1,
+                            }).then((updatedRows) => {
```
Comment (KarolineLima) [inline src/app/controller/OrderController.js]: remover then

**Review by clenia1e — APPROVED** [review]
LGTM the review

**Pull request merged**